tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -247,13 +240,15 @@ message PartitionDescProto {
   required string partitionName = 1;
   repeated PartitionKeyProto partitionKeys = 2;
   optional string path = 3;
-  optional int32 id = 4;
+  optional int32 partition_id = 4;
```

Comment:
This interface is very weird because it involves many potential bugs. We must handle the cases that some mandatory fields are missed, and this is naturally done by protocol buffer. The below problem of optionally declared path is also related to this weird design. 
A better design is as follows. PartitionDescProto corresponds to PartitionDesc, and should be used to represent a partition as TableDesc represents a table. 
To do so, PartitionDescProto should be used as only the result of the methods which retrieves partitions. For input params, a new data structure is required.